internal/models/models.go

```diff
@@ -0,0 +1,55 @@
+package models
+
+import (
+	"github.com/google/uuid"
+	"google.golang.org/protobuf/types/known/timestamppb"
+	"time"
+)
+
+type Direction string
+
+const (
+	Next     Direction = "next"
+	Previous Direction = "previous"
+)
+
+type Message struct {
+	ID         uuid.UUID `json:"id"`
+	GroupID    uuid.UUID `json:"groupId"`
+	SenderID   uuid.UUID `json:"senderId"`
+	SenderName string    `json:"senderName"`
```

Comment:
SenderName wordt uit de user service gehaald toch? Anders moet je de naam op twee plaatsen veranderen en dat is zeer bad practice

Reply:
Ja het wordt nu uit de user service gehaald en daarna opgeslagen bij elke message in de table storage. Dus je bedoelt dat ik de SenderName niet moet opslaan in de Table Storage en de user service voor elke message moet aanroepen om de senderName te krijgen wanneer er een GET voor de messages wordt gedaan? Of gewoon helemaal geen senderName terug sturen en de frontend de senderName laten requesten voor elke message? Lijkt me allebei ook niet helemaal juist.